### ISSUE TITLE
Example API: how to use Atomic Data server from Python

### ISSUE THREAD

**Issue by AlexMikhalev** [ontola/atomic-server#399]
Subj. We will be able to demonstrate the Atomic Data server and plug it into data pipeline tooling.

**Comment by joepio** [comment]
Using from python currently entails using any HTTP + JSON parsing library, I guess.

But ideally, we'd have something similar to `@tomic/lib` for Python.

Although, I'm not sure if we're talking about the same thing: I'm mostly thinking about using Atomic Server as some client, reading data and perhaps making some commits.

**Comment by AlexMikhalev** [comment]
use case: I would like to expand my notion to markdown export script to land data into atomic data server, https://alexmikhalev.medium.com/obsidian-to-notion-sync-manual-import-export-49ad429a6533 , I would like to have a doc/example where I can start.

**Comment by joepio** [comment]
I think the JSON-AD path is the easiest for this. I’ve added some pages to the docs about this. Doesn’t mention python, but should be enough for an intermediate dev to get started. What do you think?

https://docs.atomicdata.dev/create-json-ad.html

**Comment by AlexMikhalev** [comment]
Exactly, we just need to write the same calls using python-requests and write a blog posts.

**Comment by joepio** [comment]
Closing as consolidated into #1196 during the requested backlog cleanup. The remaining requirements from this issue are preserved in a new comment on #1196. This is not a claim that every requested behavior is implemented; please continue current work in that tracker. The issue and discussion remain available for reference.
